tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -326,13 +326,13 @@ install_image() {
 	fi
 
 	export AGENT_TARBALL=$(get_agent_tarball_path)
+	export AGENT_POLICY=yes
```

Comment:
Do the initrd versions need the same changes?

Reply:
Oh, they do! Let me update it.

Reply:
Updated.